pom.xml

```diff
@@ -3,79 +3,35 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.esri.geoevent.parent</groupId>
 	<artifactId>kafka</artifactId>
-	<version>10.6.0</version>
+	<version>10.7.1</version>
```

Comment:
version 10.2.0 or 10.6.0

Reply:
@hanoch I have made the requested changes in the latest commit. Thanks

Reply:
@mrasante there are more comments above

Reply:
@hanoch I have made requested changes in the above comments